src/main/java/eggventory/logic/commands/list/ListPersonCommand.java

```diff
@@ -0,0 +1,38 @@
+package eggventory.logic.commands.list;
+
+import eggventory.commons.enums.CommandType;
+import eggventory.logic.commands.Command;
+import eggventory.model.PersonList;
+import eggventory.model.StockList;
+import eggventory.storage.Storage;
+import eggventory.ui.Ui;
+
+//@@author Deculsion
+public class ListPersonCommand extends Command {
+    public ListPersonCommand(CommandType type) {
+        super(type);
+    }
+
+    /**
+     * Executes the command.
+     * @param stockList Stocklist object.
+     * @param ui ui object.
+     * @param storage storage objcet.
+     */
+    public String execute(StockList stockList, Ui ui, Storage storage) {
+        String output = "";
+
+        if (PersonList.getSize() == 0) {
+            output = "There is nobody in the list";
```

Comment:
i suspect they will say this violates slap in the sense that it is actually building assigning the output string at this point rather than just calling a method... but whatever for now